cmds/auth.go

```diff
@@ -31,11 +31,11 @@ func init() {
 }
 func authGet(c *cli.Context) error {
 	if c.NArg() == 0 {
-		return fmt.Errorf("expected one arg (id)")
+		return fmt.Errorf("Expected one arg (id)")
```

Comment:
nit, for errors strings:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Ah yes! I forget how particular Go is around errors.

Reply:
Should be better now.